server/bundler/loader.js

```diff
@@ -26,7 +26,12 @@ function getSectionsModule( sections ) {
 			} );
 		} );
 	} else {
-		dependencies = "var controller = require( 'controller' );\n";
+		dependencies = [
+			"var page = require( 'page' ),",
+			"\tcontroller = require( 'controller' ),",
+			"\preloadHub = require( 'sections-preload' ).hub;\n"
```

Comment:
is preloading even a thing you have to worry about with code-splitting turned off?